docs/design/bundle-sources.md

```diff
@@ -0,0 +1,240 @@
+# Building a file-based catalog from a plain bundle image
+
+**IMPORTANT:** Operator Lifecycle Manager (OLM) v1 features and components are still experimental. Early adopters and contributors should expect breaking changes. The following procedures are not recommended for use on production clusters.
+
+You can build a static collection of arbitrary Kubernetes manifests in the YAML format, or *plain bundle*, and add the image to a file-based catalog (FBC). The experimental `olm.bundle.mediatype` property of the `olm.bundle` schema object differentiates a plain bundle from a regular bundle. You must set the bundle media type property to `plain+v0` to specify a plain bundle.
```

Comment:
We were planning on making a follow up issue to go into the difference between a plain bundle and regular bundle. Also, I think it would be nice to include reasons why you would choose to use a plain bundle.

Do you think it is important to add that now or would a follow issue be alright?

Reply:
I think it's fine to go into the additional detail of why you would choose to use a plain bundle in a follow up and the main differences but we should at least mention that a "regular bundle" is of format `registry+v1`. Maybe something like:
```suggestion
You can build a static collection of arbitrary Kubernetes manifests in the YAML format, or *plain bundle*, and add the image to a file-based catalog (FBC). The experimental `olm.bundle.mediatype` property of the `olm.bundle` schema object differentiates a plain bundle from a regular (`registry+v1`) bundle. You must set the bundle media type property to `plain+v0` to specify a plain bundle.
```
would suffice until the follow up?

Reply:
That sounds good to me, I've addressed it based on @everettraven's suggestion in my latest commit.